README.md

```diff
@@ -42,13 +44,9 @@ npm run build
 ##### Developing:
 Run each of these commands in separate terminals:
 
-To rebuild your js every time you save:
-
+To run a development build of the project at https://localhost:3456
```

Comment:
@jsnunki I think you've mistakenly added an `https` instead of `http`, could you please update it?

Reply:
Fixed in 9e89a74eedabbeef3b4f0cd7162506594092a89d